src/Pos/Delegation/Listeners.hs

```diff
@@ -64,7 +63,9 @@ pskLightRelay = Data $ DataParams $ \pSk -> do
                logDebug $
                    sformat ("Generating delivery proof and propagating it to neighbors: "%build) pSk
                let proof = proxySign SignProxySK sk pSk pSk
-               addToRelayQueue (DataOnlyPM (pSk, proof))
+               -- FIXME seems like a mistake. Shouldn't the relay subsystem
```

Comment:
It's true that the outbound queue system is where we'd like to put most policy on which peers to send things to. That said, I don't fully understand this FIXME.